samples/ftd_configuration/anyconnect_package_file.yaml

```diff
@@ -0,0 +1,45 @@
+- hosts: all
+  connection: httpapi
+  gather_facts: no
+  vars_files:
+  - variables.yaml
+  tasks:
+  - name: Check Anyconnect Package File presense
+    ftd_configuration:
+      operation: getAnyConnectPackageFileList
+      register_as: all_anyconnect_package_files
+
+  - name: Check Anyconnect Image presense
```

Comment:
Maybe `Check AnyConnect file presence on local filesystem`? Just to clarify that we are checking the file on local filesystem, not on the FTD.

Reply:
Done